blockball-api/src/main/java/com/github/shynixn/blockball/api/business/proxy/BallProxy.kt

```diff
@@ -113,8 +123,9 @@ interface BallProxy {
      * The calculated velocity can be manipulated by the BallKickEvent.
      *
      * @param entity entity
+     * @param pass whether the kick was a pass or a shot.
      */
-    fun <E> kickByEntity(entity: E)
+    fun <E> kickByEntity(entity: E, pass: Boolean)
```

Comment:
Please create a new method "passByEntity" instead of using a boolean flag. Public contract methods need to be simple, a private method in BallProxyImpl can look like this.